Xamarin.Forms.Core/PlatformConfiguration/iOSSpecific/NavigationPage.cs

```diff
@@ -71,5 +71,28 @@ public static StatusBarTextColorMode GetStatusBarTextColorMode(this IPlatformEle
 			return config;
 		}
 		#endregion
+
+		public static readonly BindableProperty UseLargeTitlesProperty = BindableProperty.Create("UseLargeTitles", typeof(bool), typeof(Page), false);
```

Comment:
As discussed, we should probably call this `PrefersLargeTitles` to keep it in line with Apple's naming.

(We did something similar with [`PrefersStatusBarHidden`](https://github.com/xamarin/Xamarin.Forms/pull/463))